test/abstractions/providers/mturk/test_mturk.py

```diff
@@ -60,6 +60,68 @@ def test_create_and_find_worker(self) -> None:
         failed_worker = MTurkWorker.get_from_mturk_worker_id(db, "FAKE_ID")
         self.assertIsNone(failed_worker, f"Found worker {failed_worker} from a fake id")
 
+    def test_mturk_qual_create_mapping(self) -> None:
```

Comment:
Does this create enough number of request for race condition to be really tested?

Reply:
All that needs to happen to test is that `create_qualification_mapping` is called twice with the same `qualification_name`

Reply:
Yeah, but if the first one get resolve fast enough, then it doesn't happen.

Reply:
Right, the creation call doesn't do any checking, so the race occurs the moment that the system decides it will need to create a qualification. Here to be specific:
https://github.com/facebookresearch/Mephisto/blob/ce88a722729f0c6c974dcf188d9ecca82ad0e296/mephisto/abstractions/providers/mturk/mturk_worker.py#L90-L96